DirectProgramming/DPC++FPGA/include/streaming_qri.hpp

```diff
@@ -186,7 +186,7 @@ struct StreamingQRI {
 
           fpga_tools::UnrolledLoop<columns>([&](auto k) {
             auto lhs = rt_matrix[col][k];
-            auto rhs = (k == col) || (col < row) ? TT{0} :
+            auto rhs = (k >= col) || (col < row) ? TT{0} :
                                                       ri_matrix_compute[row][k];
             if(k==col){
```

Comment:
```suggestion
            if (k == col) {
```

Reply:
I'll run clang-format instead, which I forgot to run apparently
This will probably trigger many other formatting changes, but will at least be homogeneous :)